packages/cgb-scripts/package.json

```diff
@@ -55,10 +68,10 @@
 		"postcss-loader": "^2.0.10",
 		"raw-loader": "^0.5.1",
 		"resolve-pkg": "^1.0.0",
-		"sass-loader": "^6.0.6",
-		"shelljs": "^0.8.0",
+		"sass-loader": "^7.1.0",
+		"shelljs": "^0.8.3",
 		"style-loader": "^0.19.1",
 		"update-notifier": "^2.3.0",
-		"webpack": "^3.1.0"
+		"webpack": "^4.30.0"
```

Comment:
webpack 4 means that the current config will not run since the CSS Extraction is not supported the same way.